packages/core/src/lib/session.ts

```diff
@@ -41,6 +42,30 @@ export const assignInteractionResults = async (
   ctx.body = { redirectTo, ts };
 };
 
+export const checkSessionHealth = async (
+  ctx: Context,
+  provider: Provider,
+  tolerance = 10 * 60 // 10 mins
```

Comment:
For this function name "check session heath", the default tolerance should be infinite?